mysql-wordpress-pd/README.md

```diff
@@ -4,367 +4,207 @@
-[gce-volumes.yaml](https://git.k8s.io/examples/mysql-wordpress-pd/gce-volumes.yaml).
-
-```shell
-gcloud compute disks create --size=20GB --zone=<zone> wordpress-1
-gcloud compute disks create --size=20GB --zone=<zone> wordpress-2
-```
-
-Create the persistent volume objects in Kubernetes for those disks:
-
-```shell
-export KUBE_REPO=https://raw.githubusercontent.com/kubernetes/examples/master
-kubectl create -f $KUBE_REPO/mysql-wordpress-pd/gce-volumes.yaml
-```
-
-## Create the MySQL Password Secret
-
-Use a [Secret](http://kubernetes.io/docs/user-guide/secrets/) object
-to store the MySQL password. First create a file (in the same directory
-as the wordpress sample files) called
-`password.txt` and save your password in it. Make sure to not have a
-trailing newline at the end of the password. The first `tr` command
-will remove the newline if your editor added one. Then, create the
-Secret object.
-
-```shell
-tr --delete '\n' <password.txt >.strippedpassword.txt && mv .strippedpassword.txt password.txt
-kubectl create secret generic mysql-pass --from-file=password.txt
-```
-
-This secret is referenced by the MySQL and WordPress pod configuration
-so that those pods will have access to it. The MySQL pod will set the
-database password, and the WordPress pod will use the password to
-access the database.
+<!-- EXCLUDE_FROM_DOCS BEGIN -->
+
+> :warning: :warning: Follow this tutorial on the Kubernetes website:
+> https://kubernetes.io/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/.
+> Otherwise some of the URLs will not work properly.
+
+# Using Persistent Volumes with MySQL and WordPress
+<!-- EXCLUDE_FROM_DOCS END -->
+
+{% capture overview %}
+This tutorial shows you how to deploy a WordPress site and a MySQL database using Minikube. Both applications use PersistentVolumes and PersistentVolumeClaims to store data. 
+
+A [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (PV) is a set amount of storage in a cluster, and a [PeristantVolume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) (PVC) is an set amout of storage in a PV. PVs and PVCs are independent from Pod lifecycles and preserve data through restarting, rescheduling, and even deleting Pods. 
+
+**Warning:**  This deployment is not suitable for production use cases, as it uses single instance WordPress and MySQL Pods. Consider using [WordPress Helm Chart](https://github.com/kubernetes/charts/tree/master/stable/wordpress) to deploy WordPress in production.
+{: .warning}
+
+{% endcapture %}
+
+{% capture objectives %}
+* Create a PersistentVolume
+* Create a Secret
+* Deploy MySQL
+* Deploy WordPress
+* Clean up
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %} 
+
+Download the following configuration files:
+
+1. [local-volumes.yaml](https://kubernetes.io/docs/tutorials/stateful-application/local-volumes.yaml)
```

Comment:
Can you add a `mysql-wordpress-persistent-volume/` after `/stateful-application/` as I'm planning to import these files into a separate directory? (ditto for code `include`s in the rest of the file)